### DISCUSSION TITLE
VSCode - IntelliSense, debugging, and code browsing

### DISCUSSION THREAD

**Discussion by Bebert34** [community/community#40343] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,
I have been using the Arduino platform for years. I wanted to switch to VSCode. After several attempts to install (V1.73.1), I still can't get completion when typing. Also, when opening a sample project, the "c_cpp_properties.json" file does not appear in ".vscode" right away. Finally, error messages ask me to review the "includePath"! I installed the Arduino extension

Please give me some advice or steps to follow. I don't know what document to enclose, apart from the following.
Thank you
![Screen errors](https://user-images.githubusercontent.com/93608140/204510377-4108b748-1b38-4f63-a53c-22c58394c158.jpg)

**Comment by ossanna16** [comment]
Hi there @Bebert34 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
